app/utils/event.utils.ts

```diff
@@ -0,0 +1,17 @@
+import moment from 'moment';
+
+// parse event objs from backend to frontend format
+export const parseEvents = (events: any[]) =>
+  events.reduce((acc, event) => {
+    const { name, startTime, endTime, Attendees, isDeleted, ...ev } = event;
```

Comment:
`ev` could it be `remainedEventDetails` as `ev` is not providing enough information as such

Reply:
Okay 👍🏻